docs/src/openapi-v1.yaml

```diff
@@ -1589,6 +1616,10 @@ components:
           - Fee = 0
           - Misc = 1
           - All = 2
+    BlockRange:
```

Comment:
I wonder whether this should just be `Blocks`?

Reply:
So we have `Block` already and I didnt want to confuse others. But maybe the plural `Blocks` is ideal.

Reply:
Yeah; I mean it's just a nit, so I'm not bothered either way, but if I saw `Blocks` I'd instantly understand it's relationship with `Block`, whereas `BlockRange` could be something totally different.

Reply:
Yea makes sense, i'll change it up. I was kinda conflicted when writing these up anyways. I dont like the name `BlockRange` anyways.

Reply:
Fixed